### ISSUE TITLE
are tutorial code simple enough?

### ISSUE THREAD

**Issue by esaule** [BridgesUNCC/BridgesUNCC.github.io#32]
We talked about shrinking the size of the tutorial codes to really give an idea of the usage but without making the codes too long to avoid scaring the users.
Are the codes short/simple enough?

I am guessing not:
file:///home/erik/work/bridges/BridgesUNCC.github.io/tutorials/Graph_AL.html

I am wondering whether they should be straight to the point, just like what the snippet for the data sources are.

Maybe we can keep a second longer example, but maybe have a first one that is very simple?

**Comment by esaule** [comment]
I just added @AlecGoncharow as an assignee to get his opinion since he talked to a bunch of people at sigcse who had mentionned the tutorial were too complicated. Thoguht he'd have an opinion on the question.

**Comment by AlecGoncharow** [comment]
Looking at the graph code it seems we may be overwhelming people with too complex of a graph while also dipping into the data source dependent stuff. I'd recommend having a KISS example for each class/module and then having an example like the bacon number graph as an example separate from the tutorials. 
Ultimately we probably would want a barebones example for each tutorial, that way people can see the usage for each class without worrying about understanding styling logic and such.
Basically strip away the bells and whistles and just show a default graph with examples of how to add edges and such. Same could go for the linked list stuff and the arbitrary student class.

**Comment by esaule** [comment]
We've done some of that.

I suggest that we review all our existing tutorials and put explicite notes of which ones should be broken down.

**Comment by esaule** [comment]
I suppose: https://bridgesuncc.github.io/tutorials/Tree.html could be simplified by splitting structure from styling.
Same for: https://bridgesuncc.github.io/tutorials/BinTree.html

This tutorial is too complicated: https://bridgesuncc.github.io/tutorials/Graph_AM.html
rest looks fine.

**Comment by esaule** [comment]
Simplify:
- [x] https://bridgesuncc.github.io/tutorials/Tree.html 
- [ ] https://bridgesuncc.github.io/tutorials/BinTree.html
- [ ] https://bridgesuncc.github.io/tutorials/Graph_AM.html
